rhubarb/tasks.py

```diff
@@ -92,10 +93,10 @@ def extend_lock(self, ttl=None):
         """
         if ttl is None:
             ttl = self._rhubarb_settings.TASK_LOCK_TTL
-        self.__redis_client.expire(self.__lock_key, ttl)
+        self.__redis_client.expire(self.lock_key, ttl)
 
     def before_start(self, task_id, args, kwargs):
-        if not self.acquire_lock():
+        if not self.acquire_lock(kwargs.get("ttl")):
```

Comment:
nitpick: I think this would make more sense as an `__init__` param, with this implementation you can change the ttl per task execution and not per task definition

Reply:
I did some testing and was unable to get the argument in the `__init__` method, but seems that celery adds any argument to the class on initialization https://github.com/celery/celery/blob/d2aad4c650f8e742257fb0c829025058d26cc0c4/celery/app/base.py#L481

I've changed the implementation to utilize this and accept a `lock_ttl` argument on task definition